src/test/resources/MappingHelper/Custom_ListedElectronicSerial.json

```diff
@@ -9,12 +9,11 @@
       }
     },
     {
-      "field": "ACQUISITION_UNIT",
-      "dataSource": {
-        "from": "//SubAccount",
-        "default": "gobi_acq_unit_name",
-        "translation": "lookupAcquisitionUnitIdsByAccount",
-        "translateDefault": true
```

Comment:
After made this change unit test for lookupAcquisitionUnitIdsByName become to work, correct?

Reply:
yes. and the check is successful